t/file-writebytes.neon

```diff
@@ -0,0 +1,26 @@
+IMPORT file
+
+VAR buf: Bytes
+VAR bytes: Bytes
+
+buf.from_array([4, 3, 2, 0, 1])
+
+TRY
+    file.writebytes("bytetest.bin", buf)
+EXCEPTION file.FileOpenError
```

Comment:
Do you expect to get this exception? It doesn't appear so. If you aren't specifically testing the exception, then don't attempt to catch it in the test. Not catching an exception in a test will cause the test to fail. (So will writing the "write file open error" that you aren't expecting, but it's best to keep irrelevant stuff out of test cases.)